packages/cypher-builder/src/references/NodeRef.ts

```diff
@@ -25,7 +25,7 @@ import { Reference } from "./Reference";
 export type NodeProperties = Record<string, Param<any>>;
 
 type NodeRefOptions = {
-    labels?: string[];
+    labels?: Iterable<string>;
```

Comment:
Is it not better to define specifically the type as Set or Array here?
I believe that defined like that the user will not receive any errors if for instance, he passes a string like:
```ts
const node = new Cypher.Node({ 
      labels: "labels",
});
```